### ISSUE TITLE
Improve the current food and add new ones

### ISSUE THREAD

**Issue by MrJakobLaich** [GTNewHorizons/GT-New-Horizons-Modpack#8551]
### Your GTNH Discord Username

MrJakobLaich#6170

### Your Pack Version

2.1.1.0

### Your Proposal

TL;DR:

Leave healing axe as is.
Add a bunch of optional food recipes for people that want a small vacation from #automation.
I would do write of this myself. (mostly changing/adding recipes so that should be easy enough)

NOT TL;DR:

Currently food is useless once you get the healing axe.
Up until then you make wellingtons or something and don't care about anything.

I propose that we keep the healing axe as is but add more food quests for the late game.
Not like the "Ultimate Stew" but more in line with the current bread crafting steps.
This could then be a fun small side project you can do as a break from automation.
Would not produce issues with balance or anything else.
Anyone that doesn't like the addition just uses the healing axe as they would currently, so nothing changes for them either.

This would probably only add crafting recipes for food.
Also add some quests in each age that you can unlock with a checkmark-quest (no requirements, just click "I'm not using the healing axe" and you can see them.
This would allow healing axe users to not even see the quests so for them everything stays 100% the same.
The only change would be that food isn't completely useless later on in the pack.

I propose the following changes / additions (I will come up with A LOT more but that's all I have right now, also this is already too much text!!!!!!!!)
- Make different cooking tools from different materials in each age
- the higher tier foods need better tools made from higher tier materials
- change recipes for current food in a way that they match the "multi step process" of the current bread / steak sandwich recipe
-> this would mean instead of the following recipe:
![image](https://user-images.githubusercontent.com/40965742/133867160-c7b32cba-b8ef-4654-864f-30ce2172a1f0.png)
you would need to cut up the bread / cook some sandwich bread first
-> for example, the pasta currently only exists as the following recipe:
![image](https://user-images.githubusercontent.com/40965742/133867244-5e2bda89-7c38-4ae3-b53b-ec942ae595ee.png)
This change would make pasta require dough, rolled with a rolling pin, cut into the wanted lenght, hanged on the drying racks for X amount of time -> this would add multiple different kinds of pasta and make the pasta making process a bit more realistic
The next step would for example be:
![image](https://user-images.githubusercontent.com/40965742/133867367-5a8d79f2-abff-434c-bece-ab016dc88e58.png)
This just uses "pasta" + tomato + "spice leaf", I would change this so you make a dedicated sauce (different ones possible), cook the dry pasta with a pot (made in the previous step, don't forget to salt ;) ) and then you combine both to get spaghetti with sauce
Same thing with the supreme pizza, I would increase the amount of steps and also make the pizza require some kind of advanced furnace, after all you can't just make pizza by rolling dough, putting tomatoes on that and then eating it..
Supreme pizza would need some higher age tools to make.

**BEERLINE !!**

Also, it should be possible to make ceramic (add fine clay -> smelt that, not the YSZ or GDC ceramic stuff) plates and things like that, just to make the food look nicer (maybe they could be placed down as well, but that's something for later on as placing them down wouldn't be my goal right now)

So, pretty much all of this would be recipe and quest changes / additions and some aesthetic recipes like plates

Because the answer could be "who will add this", I'll say that I'll write the code myself so nobody has to waste time with this. :D

### Your Goal

Make food have a late game purpose without forcing people who don't want to use food into using food. (healing axe still exists!)

### Your Vision

Food would have a reason beyond annoying people in the earliest part of the game.
Would also be a nice _vacation_ from automation

### Final Checklist

- [X] I have searched this issue tracker and there is nothing similar already. Posting on a closed issue saying *I like this change please reconsider adding it* will prompt us to investigate and reopen it once we confirm your report.
- [X] I understand this change request may not attract enough attention and thus not be implemented.
- [X] I understand this change request may be rejected due to other community members think it's inappropriate.
- [X] I believe this feature would make the pack better.

**Comment by Prometheus0000** [comment]
> Currently food is useless once you get the healing axe.

Untrue, you get more hearts.

> Also add some quests in each age that you can unlock with a checkmark-quest (no requirements, just click "I'm not using the healing axe" and you can see them.
This would allow healing axe users to not even see the quests so for them everything stays 100% the same.
The only change would be that food isn't completely useless later on in the pack.

This is a terrible idea. Why would you take quests away from people for no reason? Also, the entire concept is bad. Again, hearts. But once you finish making all the food, yeah, it's pointless, and it won't be any less pointless if you make pointless quests for them.

> Make different cooking tools from different materials in each age
the higher tier foods need better tools made from higher tier materials

Why? This is retarded. Are you saying you need a knife made of tungstensteel to slice some bread or meat? That steel isn't good enough? Even requiring SS for a bunch of them is weird. But I believe those are non-durability versions of otherwise durability using tools.

> change recipes for current food in a way that they match the "multi step process" of the current bread / steak sandwich recipe

This would just make cooking incredibly tedious. There's no point. This isn't a cooking game.

> BEERLINE !!

Also pointless.

> Also, it should be possible to make ceramic (add fine clay -> smelt that, not the YSZ or GDC ceramic stuff) plates and things like that, just to make the food look nicer (maybe they could be placed down as well, but that's something for later on as placing them down wouldn't be my goal right now)

Making it look nicer would be done by changing the sprite, so I don't understand what you mean here. Also, you can already place things down with the DE (I think?) key.

> Make food have a late game purpose without forcing people who don't want to use food into using food. (healing axe still exists!)

This doesn't make it have a purpose though? It just makes it needlessly complicated for no reason. Again, this isn't a cooking game.

Rather than this, why don't you work with mitch to backport part/all of the nutrition mod, since that's at least useful and somewhat interesting.

**Comment by MrJakobLaich** [comment]
Thanks for the feedback, do appreciate it, please don't say that anything I come up with "is retarded" tho...

> Rather than this, why don't you work with mitch to backport part/all of the nutrition mod, since that's at least useful and somewhat interesting.

What does the nutrition mod add?

Well, regarding "it's pointless", isn't food pointless anyways because you get the healing axe?

Also, what do hearts gain you if you just one shot everything with a crossbow or are indoors 24/7 because you automate stuf

> Why would you take quests away from people for no reason?

Because, if you don't want to use the food the quests are useless for you and then showing them would be
A) cluttering stuff up
B) annoying to see quests that you 100% don't want to see

> This would just make cooking incredibly tedious. There's no point.

Pretty much everything in the pack is called "incredibly tedious" by everyone I asked except me
-> there's also no point in using the current cooking stuff because the healing axe exists...

**Comment by Prometheus0000** [comment]
The point of food is to get hearts+not die+be able to move properly (running/jumping).

> What does the nutrition mod add?

https://github.com/WesCook/Nutrition

**Comment by MrJakobLaich** [comment]
What does the nutrition mod add that would be usefull?

**Comment by Prometheus0000** [comment]
The different types? So you need to vary your food? And you get buffs of some kind.

**Comment by MrJakobLaich** [comment]
Doesn't the healing axe negate that too?

What buffs would make sense?
More damage isn't important because crossbow does enough damage I guess, correct me if I'm wrong about this
Movement speed isn't important because jetpack and stuff
hearts, maybe, how many hearts do you currently get if you eat every food?
mining speed? redstone on pickaxe...

**Comment by Prometheus0000** [comment]
Yeah, the buffs would be hard to find useful ones of. I remember another pack that just added 2 hearts for the 5 types it had.

**Comment by D-Cysteine** [comment]
Hmm, if the idea is just to give food some use, then a simple idea would be to add in a culinary generator that produces EU from food, similar to the one that Extra Utils has. As a bonus, I think this shouldn't be too much work to implement. Also, this idea doesn't preclude adding nutrition or some other use for eating food; it's more of a side-usage that we could do in addition to other changes.

Some additional ideas to flesh this out a bit:
 * We can add in a generator per tier, with decreasing efficiency at higher tiers just like combustion generators work, to keep food relevant but require better automation at higher tiers.
 * Maybe give players a low-tier generator as a quest reward for some food-related quest?
 * If we want to theme this generator to make sense, maybe have it be a Thaumcraft item? You feed food to a flesh golem that then runs on a hamster wheel or something to generate power?
 * We could assign food items to various tiers. Each subsequent tier of food would require either an additional processing step or additional difficult ingredient, justifying the higher tier. Higher tiers of food produce more base power, which is then scaled by food / saturation value to give us total power produced.
 * The generator could have multiple input slots, allowing players to give it multiple different food types. Having more unique food types gives a bonus to power generation, encouraging food variety. Higher tiers of generator could have more slots?

**Comment by Sphyix** [comment]
I like the addition of the culinary generator and the tiering of the food, i don't understand why you have to insult someone's idea and reply like that @Prometheus0000, if you don't like it, just say that, and explain why, maybe decently :)

**Comment by MrJakobLaich** [comment]
Culinary generators seem like a cool idea, sounds like that would make food a valid option late game, if the higher tier food gave more power for examle

**Comment by MrJakobLaich** [comment]
> Tiering of food would mean that you can get hearts only when getting to a tier, which makes food something that you have to keep doing if you want to get more, but instead of spamming everything at LV-MV, you carry it througout the run, a bit at a time

That's pretty what I thought too (even tho I'm not that far yet, got that info from others talking about "just get the healing axe" and "food is only annoying right now")

**Comment by Skiftcha** [comment]
remember you have bunch of food from quest rewards and dungeon loot, so quests like "make pizza" are mostly useless because you already ate it.
added sauces as separate ingredients also should be edible and add hearts. so this will also change some balance.
i like situation as is. you dont need quests for every food. you just type "not yet eaten" in NEI and cook instead of automation.

**Comment by Prometheus0000** [comment]
> I like the addition of the culinary generator and the tiering of the food, i don't understand why you have to insult someone's idea and reply like that @Prometheus0000, if you don't like it, just say that, and explain why, maybe decently :)

I didn't mean to insult anyone, I was just trying to express that I thought it was a terrible idea. I mean, it doesn't make any sense, like I stated, nor do I think it would add any value. Tiering would be better done by making the ingredients themselves only attainable at a certain stage (found on a planet or something), which I mentioned on discord, and would make way more sense. Personally I'm not a fan of adding things that have no utility basically.

For a culinary gen, I can only imagine that it would be way too easy to power, like the existing ones that are disabled in ExU. Even if you made some harder recipes, you can only make them so hard without turning off people who just want to eat them. And if you make them crazy hard to make, why not just add an actual power gen? I also question how power is gotten out of them. We can already turn organic stuff into methane and burn it, I can't imagine this would be all that different in concept, except it somehow produces way more power? Making it magic might be ok, but I imagine a hamster wheel is not going to produce more power than ULV.

Personally, I don't think it's a bad idea to be able to _finish_ food. Adding a couple super foods like jaegermeister or dezil's marshmallow would be fine, but adding a whole bunch just seems meaningless. Making it not possible to make all the best foods (if you have the ingredients) right away is an ok idea though.

**Comment by MrJakobLaich** [comment]
Fair point, don't know how food generator stuff would work either.

You mentioned making the materials only available in later stages, would adding tiers of cropsticks work? Because then the plants and everything could still exist but you can only grow for example soybeans on steel cropsticks

I can imagine this not working based on how IC2 crossbreeding works, but I don't know enough to judge that, someone tell me where I'm wrong here please

Maybe that would work well together with changing crossbreeding in a way that makes it less random? (just a thought tho)

**Comment by MrJakobLaich** [comment]
With my first sentence I mean that I don't know how the implementation would work, I'm not even close to being a pack creator (as you can tell) so idk how balancing that would work

anyways, think it's a cool idea, somehow I personally never use culinary generators though 🤔

**Comment by Prometheus0000** [comment]
IC2 is ARR and thus a giant pain in the ass to change, especially for anything complex, like adding tiers of cropsticks. Also, the more crops added, the lower the probability to get any of the existing ones. Changing crops just isn't really a big priority right now.
